pkg/meta/boltdb/boltdb.go

```diff
@@ -237,6 +295,17 @@ func (bdw *BoltDB) SetRepoReference(ctx context.Context, repo string, reference
 	return err
 }
 
+func updatedRepoLastUpdated(repo string, repoLastUpdatedBuck *bbolt.Bucket) error {
```

Comment:
updateRepoLastUpdated instead?

you can also make protoTime an arg, and also use this function in SetRepoMeta()